### ISSUE TITLE
setup.ps1 uses ARM Java binary for M1/M2/M3 Macbook

### ISSUE THREAD

**Issue by crazyhappygame** [axmolengine/axmol#1614]
**Is your feature request related to a problem? Please describe.**
setup.ps1 download and uses Java Intel binary for M1/M2/M3 Macbook

**Describe the solution you'd like**
setup.ps1 download and uses Java Intel binary for intel maccboks, ARM Java binary M1/M2/M3 Macbook

**Describe alternatives you've considered**
N/A

**Additional context**
N/A

**Comment by halx99** [comment]
m1/m2/m3 can run x86_64 exexutable file, so may not need provide arm spec tools
EDITED: And the ndk also not provide arm spec toolchain yet

**Comment by crazyhappygame** [comment]
Yes. That works now. But it is not optimal to use Intel Java binary on ARM M1/M2/M3

**Comment by halx99** [comment]
already supported
